www/include/monitoring/downtime/AddDowntime.php

```diff
@@ -319,8 +341,7 @@
     }
 
     $data["start_time"] = $centreonGMT->getDate("G:i", time(), $gmt);
-    $data["end_time"] = $centreonGMT->getDate("G:i", time() + $defaultDuration, $gmt);
-    $data["end"] = $centreonGMT->getDate("m/d/Y", time() + $defaultDuration, $gmt);
+    $data["end_time"] = $centreonGMT->getDate("G:i", time() + $defaultDuration * $durationScale[$defaultScale], $gmt);
```

Comment:
The variable shouldn't hardcode the localization format.
We add moment.js to convert BE timestamp to a localized FE one

Reply:
This end time is `GMT`, to feed the FE datepicker, which then automatically localizes it.
As the start time just above.

Reply:
Done, conversion now performed by FE.